app/Http/Controllers/FrontendUserController.php

```diff
@@ -4,6 +4,7 @@
 
 use App\Http\Controllers\UserController as UserBackend;
 use App\Models\User;
+use Carbon\Carbon;
```

Comment:
For real?! 
![download](https://user-images.githubusercontent.com/1267894/106335890-fc6f5e80-628d-11eb-9e86-eb0c220519e2.gif)